comictaggerlib/ctsettings/file.py

```diff
@@ -41,223 +40,292 @@ def internal(parser: settngs.Manager) -> None:
     )
 
 
 def talker(parser: settngs.Manager) -> None:
-    # General settings for talkers
     parser.add_setting(
         "--source",
         default="comicvine",
-        help="Use a specified source by source ID (use --list-plugins to list all sources)",
+        help="Use a specified source by source ID (use --list-plugins to list all sources).\ndefault: %(default)s",
     )
     parser.add_setting(
         "--remove-html-tables",
         default=False,
         action=argparse.BooleanOptionalAction,
         display_name="Remove HTML tables",
-        help="Removes html tables instead of converting them to text",
+        help="Removes html tables instead of converting them to text.\ndefault: %(default)s",
     )
 
 
 def md_options(parser: settngs.Manager) -> None:
     # CBL Transform settings
-    parser.add_setting("--cbl-assume-lone-credit-is-primary", default=False, action=argparse.BooleanOptionalAction)
-    parser.add_setting("--cbl-copy-characters-to-tags", default=False, action=argparse.BooleanOptionalAction)
-    parser.add_setting("--cbl-copy-teams-to-tags", default=False, action=argparse.BooleanOptionalAction)
-    parser.add_setting("--cbl-copy-locations-to-tags", default=False, action=argparse.BooleanOptionalAction)
-    parser.add_setting("--cbl-copy-storyarcs-to-tags", default=False, action=argparse.BooleanOptionalAction)
-    parser.add_setting("--cbl-copy-notes-to-comments", default=False, action=argparse.BooleanOptionalAction)
-    parser.add_setting("--cbl-copy-weblink-to-comments", default=False, action=argparse.BooleanOptionalAction)
-    parser.add_setting("--cbl-apply-transform-on-import", default=False, action=argparse.BooleanOptionalAction)
-    parser.add_setting("--cbl-apply-transform-on-bulk-operation", default=False, action=argparse.BooleanOptionalAction)
-
-    parser.add_setting("use_short_metadata_names", default=False, action=argparse.BooleanOptionalAction, cmdline=False)
-    parser.add_setting(
-        "--disable-cr",
-        default=False,
+    parser.add_setting("--assume-lone-credit-is-primary", default=False, action=argparse.BooleanOptionalAction)
+    parser.add_setting("--copy-characters-to-tags", default=False, action=argparse.BooleanOptionalAction)
+    parser.add_setting("--copy-teams-to-tags", default=False, action=argparse.BooleanOptionalAction)
+    parser.add_setting("--copy-locations-to-tags", default=False, action=argparse.BooleanOptionalAction)
+    parser.add_setting("--copy-storyarcs-to-tags", default=False, action=argparse.BooleanOptionalAction)
+    parser.add_setting("--copy-notes-to-comments", default=False, action=argparse.BooleanOptionalAction)
+    parser.add_setting("--copy-weblink-to-comments", default=False, action=argparse.BooleanOptionalAction)
+    parser.add_setting("--apply-transform-on-import", default=False, action=argparse.BooleanOptionalAction)
+    parser.add_setting("--apply-transform-on-bulk-operation", default=False, action=argparse.BooleanOptionalAction)
```

Comment:
In the GUI there is the group title `CBL Options`/`CBL Transforms` but removing the `cbl` on the cli makes it unclear these relate only to CBL, as say CIX has tags too etc. Maybe a simple `help="Relates to CBL only"` or similar?

Reply:
I'll have to double check but I do not believe it's ever actually restricted to the CBL type. Also those options are long enough as is

Reply:
I believe it is only CBL, random example from `cli.py:L353` `if self.config.Metadata_Options__apply_transform_on_bulk_operation and dst_tag_id == "cbi":`?
Although, taking the `dst_tag_id == "cbi"` out is the only reason I see.

If it's changed to all, the GUI should be changed:
![image](https://github.com/comictagger/comictagger/assets/1141189/cc9d3bee-51a9-45b2-acb7-7079e6e49aa0)

Reply:
That's for copying tags, all of the download/auto tag operations use prepare_metadata. I believe it's always been that way, but I've been wrong before and I'm not at a computer to check.

In any case the GUI needs at most a clarifying tooltip

Reply:
Even back when it was all loose files in the root directory the auto-tag would always apply it to all tags.
I added a tool tip explaining that it applies to all tags
<img width="543" alt="Screenshot 2024-06-21 at 7 09 14 PM" src="https://github.com/comictagger/comictagger/assets/7547075/236c44d3-dcc0-4ae8-b25d-497076098cc6">